dotCMS/src/main/java/com/dotmarketing/business/CacheLocator.java

```diff
@@ -406,49 +426,52 @@ public static Cachable getCache (String value) {
 	NewNotification("NewNotification Cache"),
 	VanityURLCache("Vanity URL Cache"),
 	ContentTypeCache("Legacy Content Type Cache"),
-	ContentTypeCache2("New Content Type Cache");
+	ContentTypeCache2("New Content Type Cache"),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)